src/main/java/uk/gov/hmcts/reform/translate/data/DictionaryEntity.java

```diff
@@ -29,7 +29,7 @@ public class DictionaryEntity {
     @Column(name = "translation_phrase")
     private String translationPhrase;
 
-    @OneToOne(cascade = CascadeType.PERSIST)
+    @OneToOne(cascade = CascadeType.ALL)
     @JoinColumn(name = "translation_version", referencedColumnName = "version")
```

Comment:
We are considering to add this change on [WLTS-20](https://tools.hmcts.net/jira/browse/WLTS-20)

Reply:
This has now been ajusted as described elsewhere.